pkg/cluster/spec/spec.go

```diff
@@ -510,6 +512,16 @@ func (s *Specification) ComponentsByUpdateOrder() (comps []Component) {
 	return
 }
 
+// Component implements the Topology interface
+func (s *Specification) Component(name string) Component {
```

Comment:
This repeat https://github.com/pingcap/tiup/pull/740/files#diff-196f616bff16839a1b07c563e558cb00R309